apps/astraplusplus/widget/DAO/Proposals/Card/index.jsx

```diff
@@ -493,7 +493,7 @@ const handleVote = ({
         contractName: daoId,
         methodName: isCongressDaoID ? "vote" : "act_proposal",
         args: args,
-        gas: 200000000000000
+        gas: 270000000000000
```

Comment:
Are you sure.? I m not much aware about the contract side complications with different proposal types, but I would rather have it set a bit less, if it allows almost all of the use cases.

Reply:
Most proposals need way less gas. There's no downside in attaching max because you get refund anyway.

That's why it would be best to have an input field for attached gas or maybe also linked to a slider.

Reply:
Okay I will increase it to 300Tgas. The reason I don’t want to display an input or slider in UI is most people who aren’t aware will get confused, and may be assign too little gas (in hope of saving some), and the txn might fail because of that. We can revisit it later.